ResourceStatusSystem/Service/PublisherHandler.py

```diff
@@ -34,310 +40,375 @@ def initializePublisherHandler( _serviceInfo ):
+    lastCheckTime = datetime.strptime(lastCheckTime, '%Y-%m-%d %H:%M:%S')
 
     credentials = self.getRemoteCredentials()
-    gLogger.info( credentials )
+    gLogger.info(credentials)
 
-    elementInDB =rsClient.selectStatusElement( element, 'Status', name = name,
-                                               statusType = statusType,
+    elementInDB = rsClient.selectStatusElement(element, 'Status', name=name,
+                                               statusType=statusType,
                                                # status = status,
-                                               elementType = elementType,
-                                               lastCheckTime = lastCheckTime )
-    if not elementInDB[ 'OK' ]:
+                                               elementType=elementType,
+                                               lastCheckTime=lastCheckTime)
+    if not elementInDB['OK']:
       return elementInDB
-    elif not elementInDB[ 'Value' ]:
-      return S_ERROR( 'Your selection has been modified. Please refresh.' )
-
-    reason          = 'Status %s forced by %s ( web )' % ( status, username )
-    tokenExpiration = datetime.utcnow() + timedelta( days = 1 )
-
-    newStatus = rsClient.addOrModifyStatusElement( element, 'Status', name = name,
-                                                   statusType = statusType,
-                                                   status = status,
-                                                   elementType = elementType,
-                                                   reason = reason,
-                                                   tokenOwner = username,
-                                                   tokenExpiration = tokenExpiration )
-    if not newStatus[ 'OK' ]:
+    elif not elementInDB['Value']:
+      return S_ERROR('Your selection has been modified. Please refresh.')
+
+    reason = 'Status %s forced by %s ( web )' % (status, username)
+    tokenExpiration = datetime.utcnow() + timedelta(days=1)
+
+    newStatus = rsClient.addOrModifyStatusElement(element, 'Status', name=name,
+                                                  statusType=statusType,
+                                                  status=status,
+                                                  elementType=elementType,
+                                                  reason=reason,
+                                                  tokenOwner=username,
+                                                  tokenExpiration=tokenExpiration)
+    if not newStatus['OK']:
       return newStatus
 
-    return S_OK( reason )
+    return S_OK(reason)
+
+  types_getSpaceTokenOccupancy = [(basestring, NoneType, list), (basestring, NoneType, list)]
+
+  def export_getSpaceTokenOccupancy(self, site, token):
+    """ Quite ugly way of exporting to web
+    """
+
+    endpoint2Site = {}
+
+    ses = CSHelpers.getStorageElements()
+    if not ses['OK']:
+      gLogger.error(ses['Message'])
```

Comment:
you need to return here, or something, otherwise the next line (`ses['Value']`) gives an exception